lib/open_feature/sdk/evaluation_context.rb

```diff
@@ -0,0 +1,21 @@
+module OpenFeature
+  module SDK
+    class EvaluationContext
+      TARGETING_KEY = "targeting_key"
+
+      attr_reader :fields
+
+      def initialize(targeting_key: nil, **fields)
+        @fields = {TARGETING_KEY => targeting_key}.merge(fields)
+      end
+
+      def targeting_key
+        fields[TARGETING_KEY]
+      end
+
+      def field(key)
+        fields[key]
+      end
```

Comment:
I started with implementing the `method_missing` idiom here to delegate all method calls to the underlying fields hash. I decided to defer that since this implementation satisfies the spec and only have `targeting_key` be a convenience method added for now. Definitely open to adding it in the future.